base/uk.ac.stfc.isis.ibex.epics/src/uk/ac/stfc/isis/ibex/epics/conversion/VTypeDefaultFormatter.java

```diff
@@ -59,8 +73,10 @@ public String convert(VType value) throws ConversionException {
 		}
 	}
 	
-	public String asString(VNumber vnum) {
-		return vnum.getFormat().format(vnum.getValue());
+    private String asString(VNumber vnum) {
+        NumberFormat vnumFormat = vnum.getFormat();
+        NumberFormat exponentialWhenNeededFormat = new ExponentialOnThresholdFormat(vnumFormat);
+        return exponentialWhenNeededFormat.format(vnum.getValue());
```

Comment:
Could be done as:

```
return (new ExponentialOnThresholdFormat(vnum.getFormat())).format(vnum.getValue());
```

Personally I don't think this reduces readability given the already long variable names. At the very least I think `vnumFormat` is redundant and can be replaced directly with `vnum.getFormat()`